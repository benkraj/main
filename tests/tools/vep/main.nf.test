nextflow_pipeline {

    name "Test --tools vep"
    script "main.nf"

    test("Should run without failures") {

        when {
            params {
                outdir = "$outputDir"
                test = true
                tools = 'vep'
                genome = 'WBcel235'
            }
        }

        then {
            assert workflow.success
            assert new File("$outputDir/ensemblvep/test_vcf_gz/test_VEP.ann.summary.html").exists()
            assert new File("$outputDir/ensemblvep/test_vcf_gz/test_VEP.ann.vcf.gz").exists()

            assert snapshot(
                "$outputDir/ensemblvep/versions.yml"
            ).match()
        }
    }
}
