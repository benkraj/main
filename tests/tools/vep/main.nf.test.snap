{
    "Should run without failures": {
        "content": [
            "versions.yml:md5,9c396a3750ef949fbfdfa0b40ca5bc15"
        ],
        "timestamp": "2023-02-28T16:49:27+0000"
    }
}